app/experimenter/static/visualization/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@experimenter/visualization",
+  "license": "MPL-2.0",
+  "version": "0.0.0",
+  "scripts": {
+    "build": "npm run clean; rollup --config",
+    "clean": "rimraf ../assets/visualization",
+    "watch": "rollup --config --watch",
+    "format": "prettier --plugin-search-dir=. --write .",
+    "format:check": "prettier --plugin-search-dir=. --check .",
+    "lint": "eslint --ext .js .; svelte-check",
+    "lint-fix": "eslint --fix --ext .js, .;"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.9.0",
+    "@graph-paper/core": "0.0.0-alpha.12",
+    "@rollup/plugin-commonjs": "^11.0.0",
+    "@rollup/plugin-node-resolve": "^7.0.0",
+    "@rollup/plugin-typescript": "^5.0.2",
+    "@tsconfig/svelte": "^1.0.10",
+    "@typescript-eslint/eslint-plugin": "^4.1.0",
+    "@typescript-eslint/parser": "^4.1.0",
+    "eslint": "^7.8.1",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-prettier": "^3.1.3",
+    "eslint-plugin-svelte3": "^2.7.3",
+    "prettier": "^2.0.5",
+    "prismjs": "^1.20.0",
+    "rimraf": "3.0.2",
```

Comment:
Seems to be the only devDep without a semver range operator (`^`).
I know we're planning a refactor of the nested app/experimenter/ directories, but we should also make sure that dependabot also picks up on this package.json file.

Reply:
Yeah good call @pdehaan , I think doing this change will make dependabot pick up on all the frontend dependencies with a single config: https://github.com/mozilla/experimenter/issues/3399 so we can probably skip adding a specific dependabot config here.